lib/index.js

```diff
@@ -160,12 +170,15 @@ function UseCmdLine (parsedArgs) {
 // Prints out some minor help text
 function PrintHelp () {
   return `
-Usage: env-cmd [env_file | env_name] command [command options]
+Usage: env-cmd [option] [env_file | env_name] command [command options]
 
 A simple utility for running a cli application using an env config file.
 
 Also supports using a .env-cmdrc json file in the execution directory to support multiple
 environment configs in one file.
+
+Options:
+  --no-override - if you do not wath shell envs to be overrided by env_file
```

Comment:
`  --no-override - do not override existing process env vars with file env vars`